src/node/feature.ts

```diff
@@ -0,0 +1,29 @@
+export enum Feature {
+  /** Web socket compression. */
+  PermessageDeflate = "permessage-deflate",
+}
+
+const enabledFeatures: Feature[] = []
+
+/**
+ * Enable the specified feature.
+ *
+ * Return true if the feature was enabled or false if it's invalid.
+ */
+export function enableFeature(feature: string) {
+  for (const key in Feature) {
+    if (Feature[key as keyof typeof Feature] === feature) {
+      enabledFeatures.push(feature as Feature)
+      return true
+    }
+  }
+  return false
+}
+
+/**
+ * Return true if the feature is enabled. Features are always enabled in
+ * development mode to ensure we are testing them.
+ */
+export function isFeatureEnabled(feature: Feature) {
+  return enabledFeatures.includes(feature)
+}
```

Comment:
This looks great and ++ for the documentation! 🎉 

nit: we may want to modify these functions to take `enabledFeatures` as an arg. That way, they're not relying and a global value, which could lead to unintended consequences later. Plus, it helps with testing.

Reply:
Hmmm, yeah if we made `enabledFeatures`  immutable then we'd get purity.

Buuuut having to pass it around everywhere we need it seems a bit tiresome. :stuck_out_tongue: 

It also seems to create a bit of a mismatch. Feature flags are global settings but if you can pass them in it creates the illusion that they aren't. So the use and the design don't quite match up.

What kind of unintended consequences you were thinking of?

Reply:
Actually...this is all entirely unnecessary lol. I'll just leave the enum then a caller can do `req.args.enable.includes(Feature.MyFeature)`.